modules/social_features/social_event/social_event.module

```diff
@@ -1004,10 +1074,12 @@ function social_event_preprocess_field(&$variables) {
  * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
  */
 function _social_event_get_enrollment_status(NodeInterface $event) {
-  $enrollments = \Drupal::entityTypeManager()->getStorage('event_enrollment')->loadByProperties([
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.